src/include/network/network_defs.h

```diff
@@ -20,15 +20,21 @@
 #include "common/macros.h"
 #include "loggers/main_logger.h"
 #include "parser/pg_trigger.h"
+#include "traffic_cop/result_set.h"
 #include "type/type_id.h"
 
+namespace terrier::traffic_cop {
+class TrafficCop;
+}
+
 namespace terrier::network {
+class PostgresPacketWriter;
 
 // For epoch
 // static const size_t EPOCH_LENGTH = 40;
 
 // For threads
-#define CONNECTION_THREAD_COUNT 1
+#define CONNECTION_THREAD_COUNT 11
```

Comment:
11 😐

Reply:
Did we say that this should be 4 for now?